CRBM-Viz/src/app/Services/auth0.service.ts

```diff
@@ -44,6 +44,23 @@ export class AuthService {
   // Create subject and public observable of user profile data
   private userProfileSubject$ = new BehaviorSubject<any>(null);
   userProfile$ = this.userProfileSubject$.asObservable();
+  extendedProfile: object = new Object({
```

Comment:
The type of this object must be specified in a model and used here. The object type wont have any of the feilds added here, leading to the compile error currently

Reply:
That makes sense. Having not ready any of the documentation for Angular, I'm slowly learning.

You could leave this out until I create a model.